src/app/components/person/person.component.ts

```diff
@@ -11,7 +11,10 @@ import { LoaderService } from '../partials/spinner/loader.service';
 @Component({
   selector: 'app-person',
   templateUrl: './person.component.html',
-  styleUrls: ['./person.component.scss']
+  styleUrls: ['./person.component.scss'],
+  // disabled for styling innerHTML code
+  // tslint:disable-next-line:use-component-view-encapsulation
+  encapsulation: ViewEncapsulation.None
```

Comment:
What's wrong with innerHTML? Why we need this?

Reply:
From the backend  we get innerHTML code and in result we have some issues for styling it. One of them - styling tag `<a>`. Of course, we can change styles  for all tags like that in main component, but it will be global changes and need a lot of time for testing. I note this case and I will resolve it with code refactoring.